src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs

```diff
@@ -59,7 +64,7 @@ public string FullScript
                         identity != null &&
                         identity.IsAuthenticated)
                     {
-                        string escapedUserName = identity.Name.Replace("\\", "\\\\");
+                        string escapedUserName = encoder.Encode(identity.Name);
```

Comment:
I'm not sure if we duplicated it on GitHub, it was a security bug from the SDL review.